workshop/content/docs/api-deep-dive/sensorthings.md

```diff
@@ -208,6 +208,34 @@ http://toronto-bike-snapshot.sensorup.com/v1.0
 The link to the request
 is: <http://toronto-bike-snapshot.sensorup.com/v1.0/Datastreams(206051)/Observations(1593917)?$select=result>
 
+Checkout various available public end points [here](https://github.com/opengeospatial/sensorthings/blob/master/PublicEndPoints.md)
+#### QGIS Implementation 
```

Comment:
@krishnaglodha thanks for adding the section about QGIS usage. Please move it from here to the demo section, on [this](https://github.com/opengeospatial/ogcapi-workshop/blob/main/workshop/content/docs/api-deep-dive/sensorthings.md?plain=1#L633) placeholder.